### ISSUE TITLE
JsonSyntaxException MalformedJsonException from AutoConfigManager.load

### ISSUE THREAD

**Issue by keturn** [MovingBlocks/Terasology#4642]
### What you were trying to do

Start the game.

### What actually happened

Before getting to the main menu, the game crashes with 

```
[main] ERROR o.t.engine.core.TerasologyEngine - Failed to initialise Terasology
com.google.gson.JsonSyntaxException: com.google.gson.stream.MalformedJsonException: Use JsonReader.setLenient(true) to accept malformed JSON at line 1 column 4 path $
        at com.google.gson.Gson.assertFullConsumption(Gson.java:908)
        at com.google.gson.Gson.fromJson(Gson.java:871)
        at org.terasology.engine.persistence.typeHandling.gson.GsonPersistedDataReader.read(GsonPersistedDataReader.java:33)
        at org.terasology.engine.persistence.typeHandling.gson.GsonPersistedDataReader.read(GsonPersistedDataReader.java:16)
        at org.terasology.persistence.serializers.Serializer.deserialize(Serializer.java:84)
        at org.terasology.engine.config.flexible.AutoConfigManager.loadSettingsFromDisk(AutoConfigManager.java:81)
```

there may be other errors shown adjacent to this as things crash.

There's a config file that contains
```json
{}debugEnabled":true}
```

which looks like something probably opened an existing file without truncating it and wrote the empty object `{}`. 

### Log details and game version

running develop sometime after 4.3.0

**Comment by keturn** [comment]
this is a thing that should get improved error reporting (_which_ file had the syntax error?) as well as a fix.

**Comment by keturn** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Write up issue for JSON issue](https://trello.com/c/Gfaph0HA/103-write-up-issue-for-json-issue)

**Comment by DarkWeird** [comment]
My error.
I did use invalid OpenOptions for config files.
fixed at #4660
